### ISSUE TITLE
training loss error: inf 

### ISSUE THREAD

**Issue by jxlijunhao** [lzx1413/PytorchSSD#2]
I follow the steps in the readme, but it occurs trainning loss error, the trainning locatization loss
the logs:

  1 Epoch:1 || epochiter: 0/517|| Totel iter 0 || L: inf C: 23.2587||Batch time: 16.6584 sec. ||LR: 0.00000100                                 
  2 Epoch:1 || epochiter: 10/517|| Totel iter 10 || L: inf C: 19.0657||Batch time: 0.8695 sec. ||LR: 0.00001647
  3 Epoch:1 || epochiter: 20/517|| Totel iter 20 || L: inf C: 16.3741||Batch time: 0.9224 sec. ||LR: 0.00003194
  4 Epoch:1 || epochiter: 30/517|| Totel iter 30 || L: inf C: 14.5374||Batch time: 0.8792 sec. ||LR: 0.00004741

**Comment by lzx1413** [comment]
which model are you training?

**Comment by manlinting** [comment]
I encountered the same problem with RFB_mobile 300.

**Comment by lzx1413** [comment]
I will update the mobilenet models without pyinn in a few days, please stay tuned.

**Comment by manlinting** [comment]
I tested the other network, the same problem. Is it related to my use of python2.7?

**Comment by lzx1413** [comment]
It only support python3 now, the prior box generation needs the division in python3, you may  solve this problem with using from __future__ import division.

**Comment by manlinting** [comment]
after from __future__ import division, I have solve this problem.thank you very much! Another question, can you help to  support mobilenetv2?

**Comment by lzx1413** [comment]
@manlinting https://github.com/ShuangXieIrene/ssds.pytorch there is a another pytorch ssd including mobilentv2

**Comment by manlinting** [comment]
thanks!
